docs/cmdline-opts/haproxy-clientip.d

```diff
@@ -0,0 +1,16 @@
+c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+SPDX-License-Identifier: curl
+Long: haproxy-clientip
+Help: Sets client IP in HAProxy PROXY protocol v1 header
+Protocols: HTTP
+Added: 8.1.0
+Category: http proxy
+Example: --haproxy-clientip $IP
+See-also: proxy
+Multi: boolean
+---
+Sets a client IP in HAProxy PROXY protocol v1 header at the beginning of the
+connection.
```

Comment:
Maybe add some more words on how that IP address should be phrased, like syntax etc? What happens if I provide a bad IP ? Does it support IPv6?

It's not a `Multi: boolean` is it? It sets a string, I believe `single`?

Maybe mention that it *replaces* `--haproxy-protocol` when used? Ie no need to use both.

Reply:
Will d